css/style.css

```diff
@@ -0,0 +1,93 @@
+a {
+  text-decoration: none;
+}
+
+ul {
+  
+  margin-bottom: 9px;
+  background: gray;
+
+}
+.active {
+  background-color: rgb(255, 255, 255);
+  color: gray;
+}
+li {
+  display: inline;
+  text-decoration: none;
+  padding-left: 80px;
+  padding-bottom: 20px;
+
+}
+
+#logo {
+  margin-left: 200px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.nav-item {
+  margin-top: 5px;
+  margin-bottom: 20px;
+  
+}
+
+main {
+  width: 960px;
+  margin: auto;
+  overflow: auto;
+}
+
+footer {

```

Comment:
You should style things in the order they appear in the HTML to help you keep your CSS more organized. The footer, since it's at the end of the HTML document, should be styled at the end of the CSS file.